_posts/2018-08-20-announcing-bitcoin-abc-0-18-0.md

```diff
@@ -0,0 +1,37 @@
+---
+layout: post
+title: Announcing Bitcoin ABC 0.18.0
+subtitle: Be sure to upgrade before the November upgrade
+---
+
+We are pleased to announce that Bitcoin ABC 0.18.0 has been released. You can download it at: 
+[https://download.bitcoinabc.org/0.18.0/](https://download.bitcoinabc.org/0.18.0/).
+
+The goal of Bitcoin ABC is to help create sound money that is usable by everyone in the world.
+We believe this is a civilization-changing technology which will dramatically increase human freedom
+and prosperity. Bitcoin ABC 0.18.0 represents the next step towards this goal
+
+Bitcoin ABC 0.18.0 includes implementation and activation of the
+[Bitcoin Cash network upgrade](https://github.com/bitcoincashorg/bitcoincash.org/pull/94/files)
+that will take effect on Nov 15, 2018. All users of Bitcoin ABC should upgrade to version 0.18.0
+as soon as possible.
+
+We aim to work in the best interests of miners, investors, and users of Bitcoin Cash, and to collaborate
+and coordinate with other teams and projects in the Bitcoin Cash ecosystem. Bitcoin ABC 0.18.0 grew out of
+this collaborative process. It implements network upgrade features developed with input and collaboration
+from many different groups. It is an important step in the process of careful and incremental technical
+progress to improve the utility of Bitcoin Cash and enable massive on-chain scaling.
+
+The specific features included in the November 15th upgrade, and included in Bitcoin ABC 0.18.0, include:
```

Comment:
I think this part should change because "include" is used 3 times in the same sentence.
and included in Bitcoin ABC 0.18.0...
and provided by Bitcoin ABC 0.18.0...

Reply:
OK, fixed